ce/tests/test_api.py

```diff
@@ -168,28 +183,67 @@ def test_data_bad_time(populateddb):
     assert 'time parameter "time-not-an-int" not convertable to an integer.' == \
         str(exc.value)
 
-@pytest.mark.parametrize(('variable'), ('tasmax', 'tasmin'))
-def test_data_single_variable_file(populateddb, variable):
-    rv = data(populateddb.session, 'cgcm3', 'rcp45', 1, None, variable,
-              timescale='monthly')
+
+@pytest.mark.parametrize('variable', (
+        'tasmax',
+        'tasmin',
+))
+@pytest.mark.parametrize('timescale, time_idx, expected_ymd', (
+        ('monthly', 8, (1985, 9, 15)),
+        ('seasonal', 2, (1985, 7, 15)),
+        ('yearly', 0, (1985, 7, 2)),
+))
+def test_data_single_file(populateddb, variable,
```

Comment:
[Here's what happens at present](http://docker2.pcic.uvic.ca:8300/api/data?ensemble_name=downscaled&model=bcc-csm1-1-m&variable=tasmax&emission=historical,+rcp45&timescale=monthly&area=&time=12). 

The frontend data validation functions [check to see](https://github.com/pacificclimate/climate-explorer-frontend/blob/master/src/components/DataControllerMixin/DataControllerMixin.js#L93) if the backend returns a string object instead of a JSON object to catch that sort of thing.

There may be a better way to deal with this, but that's what happens now.